### PR TITLE
Fix inplace success message

### PR DESCRIPTION
**Pull request by thatch**
It's a little unfortunate the branch messages are in `runner.py` and the inplace ones are here.  Long term we should unify that.

### CODE REVIEW

**Review by amjith — APPROVED** [review]

**Comment by amjith** [comment]
Weird. It is passing locally. I'll see what is different in GH actions.

File: advice_animal/cli.py [hunk 1]
```diff
@@ -273,13 +284,22 @@ def apply(ctx: click.Context, target: str, inplace: bool) -> None:
...
+                click.echo(
+                    click.style(advice_name, fg="green")
+                    + ": "
+                    + (result.message or "Changes made")
+                )
                 for next_step in result.next_steps:
                     click.echo(click.style(advice_name, fg="yellow") + ": " + next_step)
             else:
                 click.echo(click.style(advice_name, fg="green") + ": No changes needed")
+            return True
```
Comment (thatch) [inline advice_animal/cli.py:296]: this line seems wrong

Comment (thatch) [reply advice_animal/cli.py:296]: seems like it will only ever run one advice

Comment (amjith) [reply advice_animal/cli.py:296]: It return True if the result.success is True. What seems wrong?

**Pull request merged**